go/mysql/flavor.go

```diff
@@ -32,7 +32,7 @@ import (
 var (
 	// ErrNotReplica means there is no replication status.
 	// Returned by ShowReplicationStatus().
-	ErrNotReplica = errors.New("no replication status")
+	ErrNotReplica = NewSQLError(ERNotReplica, SSUnknownSQLState, "no replication status")
 
 	// ErrNoPrimaryStatus means no status was returned by ShowPrimaryStatus().
 	ErrNoPrimaryStatus = errors.New("no master status")
```

Comment:
Should we invent an error code for this and make this a SQLError too?

Reply:
For now, we are only returning that error and not checking for it anywhere which is what the motivation to add the code for the error was in the first place. If you want, I can add it to this error too, but it won't be used anywhere except for returning this error